cql3/expr/expression.hh

```diff
@@ -22,6 +22,7 @@
 #pragma once
 
 #include <fmt/core.h>
+#include <ostream>
```

Comment:
I think <iosfwd> is sufficient here.

Reply:
Sorry, I don't understand what you meant by this?